server/api/jellyfin.ts

```diff
@@ -0,0 +1,246 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import axios, { AxiosInstance } from 'axios';
+import logger from '../logger';
+
+export interface JellyfinUserResponse {
+  Name: string;
+  ServerId: string;
+  ServerName: string;
+  Id: string;
+  PrimaryImageTag?: string;
+}
+
+export interface JellyfinLoginResponse {
+  User: JellyfinUserResponse;
+  AccessToken: string;
+}
+export interface JellyfinLibrary {
```

Comment:
Can we be consistent with line breaks in this file?  I would add line breaks between the interfaces/classes.  You currently have for some but not all 😛